pkg/util/util.go

```diff
@@ -62,6 +62,25 @@ func GetAzureUser() (string, error) {
 	return v.UserName, nil
 }
 
+// GetAzureEndpoint returns the Azure endpoint.
+func GetAzureEndpoint() (string, error) {
```

Comment:
let's move off of this model this fragile, see my other comment about adding a setting.